app/screens/network/Network.tsx

```diff
@@ -323,16 +323,12 @@ const Network = ({ history }) => {
                     text="REDIST INSTALLATION GUIDE."
                   />
                 )}
-                {(isShowMissingLibsMessage ||
-                  nodeError?.type === NodeErrorType.NOT_SPECIFIED) && (
-                  <ErrorMessage compact>
-                    WE HAVE A CHECKLIST THAT CAN HELP YOU DETECT THE ISSUE.{' '}
-                    <Link
-                      style={{ display: 'inline-block' }}
-                      onClick={() => setOpenCheckListModal(true)}
-                      text="OPEN CHECKLIST."
-                    />
-                  </ErrorMessage>
+                {nodeError?.type === NodeErrorType.NOT_SPECIFIED && (
```

Comment:
<img width="968" alt="Screenshot 2023-11-21 at 01 00 14" src="https://github.com/spacemeshos/smapp/assets/54288612/afdcd71d-c86f-4b64-b24d-b778185fcfe5">

Additionally, for longer messages, consider adding a tooltip to accommodate the extra text. This is not critical, but it might still be unclear to the community whether it's better with or without the tooltip. However, based on the screenshots that will be posted on Discord, it might be easier to identify any issues.

Reply:
Oki. Actually, there is enough space to display 5 lines of text. So I will make it show up to 5 lines.
And actually, I think there should not be more than 3-4 lines :)

Putting the rest in the tooltip does not sound really helpful.